android/library/src/main/java/com/polidea/multiplatformbleadapter/errors/BleError.java

```diff
@@ -1,7 +1,7 @@
 package com.polidea.multiplatformbleadapter.errors;
 
 
-public class BleError {
+public class BleError extends Throwable {
```

Comment:
I wonder if this should be called BleError and extend throwable, or be about an exception?
https://stackoverflow.com/questions/912334/differences-between-exception-and-error